README.md

```diff
@@ -584,6 +584,26 @@ Browsershot::url('https://example.com')
     ->save($pathToImage);
 ```
 
+#### Changing the value of a dropdown value
+
+You can change the value of a dropdown on the page (you can use this to change form select fields).
+
+```php
+Browsershot::url('https://example.com')
+    ->dropdown_select('#selector1', '100')
+```
+
+You can combine `dropdown_select`, `type` and `click` to create a screenshot of a page after submitting a form:
+
+```php
+Browsershot::url('https://example.com')
+    ->type('#firstName', 'My name')
+    ->dropdown_select('#state', 'MT')
```

Comment:
rename the method to `select`

Reply:
There is already a method named [select](https://github.com/spatie/browsershot/blob/master/src/Browsershot.php#L215) I'm not in love with dropdown_select, but couldn't think of a good descriptive name.

Reply:
`select_option`?

Reply:
`selectOption` sounds good

Reply:
Thanks! It is done!